### DISCUSSION TITLE
Godot 4

### DISCUSSION THREAD

**Discussion by GusRD07** [community/community#109495] [Programming Help]
### Select Topic Area

Question

### Body

Hola, alguien sabe como pasar un código en python para que sea compatible con Godot 4? Que tengo un programa hecho de un juego y me molaría convertirlo en un juego de verdad ahora. Gracias.

**Comment by queenofcorgis** [comment]
Hi @GusRD07 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
